screener-e2e/examples/test/configs/wdio.shared.conf.js

```diff
@@ -0,0 +1,33 @@
+exports.config = {
```

Comment:
Why do you have this file if you also have a separate local config that doesn't re-use this file. Or re-use this file for also the local one, or remove this one and add this in the `wdio.screener.saucelabs.conf.js`